src/components/AllArticles.jsx

```diff
@@ -0,0 +1,53 @@
+import Dropdown from "react-bootstrap/Dropdown";
+import { ArticleCard } from "./ArticleCard.jsx";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export const AllArticles = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setisLoading] = useState(true);
+
+  useEffect(() => {
+    axios
+      .get("https://news-api-40x5.onrender.com/api/articles")
+      .then(({ data: { articles } }) => {
+        setArticles(articles);
+        setisLoading(false);
+      });
+  }, []);
+
+  return (
+    <div className={"all-articles"}>
```

Comment:
This could certainly be more accessible and use some more semantic tags. Currently there are a lot of divs and not much else to help those using a screen reader as you have with the footer and header